### ISSUE TITLE
Add the possibility to run binaries instead of libfuzzer harnesses

### ISSUE THREAD

**Issue by jvoisin** [google/oss-fuzz#8818]
Some software (like `tar`, `jq`, `vim`, ...) are ~~too cursed~~ not designed as a library and thus can't easily (or even at all) be shoehorned into a libfuzzer harness, yet are not only used a lot but are prime targets of some [popular exploit acquisition programs]( https://zerodium.com/program.html ).

It would be nice to be able to fuzz them :)

**Comment by jonathanmetzman** [comment]
Related #5902

**Comment by jonathanmetzman** [comment]
David, is there a PR on the CF side? Or do I need to implement this?

**Comment by DavidKorczynski** [comment]
> Or do I need to implement this?

Yes -- I'll fix the issues in the PR and get that ready for merge on the OSS-Fuzz side.
